### ISSUE TITLE
Question: is every game now listed as revive dashboard normal?

### ISSUE THREAD

**Issue by Theoneonlykid** [LibreVR/Revive#723]
as the title states my steam vr states im playing revivve dashboard is that normal now with the new update or is something wrong with the way my revive install and steam are communicating?

**Comment by jaorg1234** [comment]
It seems like the Oculus update 1.17 adds other VR apps to the Revive dashboard if they have been launched while the Oculus services are running.

E.g. here you can see UE4 got added, but no other SteamVR apps appear in the Revive dashboard: http://imgur.com/zFjnc3h

Usually I turn off the Oculus services, but only once had them turned on when opening up UE4. Right afterwards UE4 was in the Revive dashboard.

**Comment by Armada651** [comment]
Duplicate of #706
